reppy/parser.py

```diff
@@ -149,6 +150,8 @@ def _regex_rule(rule):
         # '*' and '/'
         if rule and rule[0] != '/' and rule[0] != '*':
             rule = '/' + rule
+        # collapse many **** into a single one
+        rule = RE_ASTERISK.sub('*', rule)
         tmp = re.escape(unquote(rule.replace('%2f', '%252f')))
         return re.compile(tmp.replace('\*', '.*').replace('\$', '$'))
```

Comment:
Alternatively, we could just do `tmp.replace('\*+', '.*')`, but this looks fine to me